fusetrace/fusetrace_ll.cc

```diff
@@ -81,10 +81,12 @@ struct Inode {
 
     Inode() : ref(0) {}
 
-    Inode *lookup(const string& dname) { 
-	if (dentries.count(dname))
-	    return dentries[dname];
-	return 0;
+    Inode *lookup(const string& dname)
+    {
+        auto dentry = dentries.find(dname);
+	    if (dentry != dentries.end())
```

Comment:
@rishabhkumar296

please note that we use tab-stop of 8. and these two line are not aligned: there are 8 spaces before `auto`, and 1 tab and 4 spaces before `if`, which makes the `if` block is wrongly indented.